resource_booking/models/resource_booking.py

```diff
@@ -532,29 +532,43 @@ def name_get(self):
             new.append((id_, name))
         return new
 
+    def _get_combination_from_updated_values(self, updated_values):
+        if updated_values.get("combination_id"):
+            return (
+                self.env["resource.booking.combination"]
+                .sudo()
+                .browse(updated_values.get("combination_id"))
+            )
+        if updated_values.get("meeting_id"):
+            event = (
+                self.env["calendar.event"]
+                .sudo()
+                .browse(updated_values.get("meeting_id"))
+            )
+            booking = fields.first(event.resource_booking_ids)
+            return booking.filtered(
+                lambda x: not x.combination_auto_assign and x.combination_id
+            ).combination_id
+        return False
+
     def _message_auto_subscribe_followers(self, updated_values, default_subtype_ids):
         """Auto-subscribe and notify resource partners."""
         result = super()._message_auto_subscribe_followers(
             updated_values, default_subtype_ids
         )
-        combination = (
-            self.env["resource.booking.combination"]
-            .sudo()
-            .browse(updated_values.get("combination_id"))
-        )
-        resource_partners = combination.mapped(
-            "resource_ids.user_id.partner_id"
-        ).filtered("active")
-        for partner in resource_partners:
-            result.append(
-                (
-                    partner.id,
-                    default_subtype_ids,
-                    "resource_booking.message_combination_assigned"
-                    if partner != self.env.user.partner_id
-                    else False,
+        combination = self._get_combination_from_updated_values(updated_values)
+        if combination:
+            resource_partners = combination.mapped("resource_ids.user_id.partner_id")
+            for partner in resource_partners.filtered("active"):
+                result.append(
```

Comment:
Doing this they receive the booking details? What if they aren't subscribed by default to notes?